apps/settings/js/panels/developer/developer.js

```diff
@@ -96,19 +99,25 @@ define(function(require) {
      * @access private
      * @memberOf Developer.prototype
      */
-    _resetDevice: function d__resetDevice() {
+    _resetDevice: function d__resetDevice(event) {
```

Comment:
Let's not pass the event object to this function as it is not related to the function.